website/src/assets/styles/tailwind.css

```diff
@@ -50,6 +50,12 @@ limitations under the License.
     --ring: 24.6 95% 53.1%;
     --radius: 0.5rem;
 
+    --chart-1: 12 76% 61%;
+    --chart-2: 173 58% 39%;
+    --chart-3: 197 37% 24%;
+    --chart-4: 43 74% 66%;
+    --chart-5: 27 87% 67%;
```

Comment:
Let's rename these (along with the ones in the dark theme) so they are easily understandable.

Reply:
I'll rename this on the daily page as i'll use those variables for the complete charts